### PR TITLE
Add sensor slider example

### PR DESCRIPTION
**Pull request by kacperkapusciak**
🐶

### CODE REVIEW

**Review by tjzel — APPROVED** [review]
🚀

File: app/src/examples/VolumeExample.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    }
+    const ax = animatedSensor.sensor.value.x * 0.0001;
+    vx.value += ax * dt;
+    x.value += vx.value * dt;
+    x.value = Math.min(100, Math.max(0, x.value));
+
+    if (x.value === 0 || x.value === 100) {
+      vx.value = 0;
+    }
+  });
```
Comment (piaskowyk) [inline app/src/examples/VolumeExample.tsx:69]: Just wondering 🤔 Is a frame callback necessary here? Maybe the better idea would be the usage of the `useDerivedValue` or `useAnimationReaction` hook here.

Comment (kacperkapusciak) [reply app/src/examples/VolumeExample.tsx:69]: Hm 🤔 I kinda need the change in time `dt` for physics calculations

File: app/src/examples/VolumeExample.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+} from 'react-native';
+import React, { useLayoutEffect } from 'react';
+import { useNavigation } from '@react-navigation/native';
+import Slider from '@react-native-community/slider';
+
+const primaryColor = '#FFD61E';
+const secondaryColor = '#f5f5f5';
+const backgroundColor = '#232736';
+const secondaryBackgroundColor = '#1B2445';
+
```
Comment (tomekzaw) [inline app/src/examples/VolumeExample.tsx:25]: plz unify lowercase/uppercase

File: app/src/examples/VolumeExample.tsx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    return {
+      value: x.value,
+    };
+  });
+
+  const animatedTextProps = useAnimatedProps(() => {
+    return {
+      text: String(Math.floor(x.value)),
+      // Here we use any because the text prop is not available in the type
+    } as any;
```
Comment (tomekzaw) [inline app/src/examples/VolumeExample.tsx:81]: Can you please double-check if that's still valid?

File: app/src/examples/VolumeExample.tsx [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    };
+  });
+
+  return (
+    <View
+      style={{
+        flex: 1,
+        flexDirection: 'column',
+        backgroundColor,
+        padding: 20,
```
Comment (tomekzaw) [inline app/src/examples/VolumeExample.tsx:96]: Can you please static styles to `styles`?

File: app/src/examples/VolumeExample.tsx [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+  },
+  header: {
+    fontSize: 24,
+    color: secondaryColor,
+    fontFamily: 'Poppins-Bold',
+  },
+  subHeader: {
+    fontSize: 24,
+    color: secondaryColor,
+    fontFamily: 'Poppins-Bold',
```
Comment (tomekzaw) [inline app/src/examples/VolumeExample.tsx:209]: Do you think if it makes sense to have `const FONT_FAMILY = 'Poppins-Bold';` and use `FONT_FAMILY` everywhere instead?

File: app/src/examples/VolumeExample.tsx [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+          value={true}
+        />
+      </View>
+      <View style={styles.divider} />
+      <Text style={[styles.header, styles.margin]}>Streaming quality</Text>
+      <Text style={styles.text}>Automatic</Text>
+      <Text style={styles.text}>Low</Text>
+      <Text style={styles.text}>Normal</Text>
+      <Text style={styles.text}>High</Text>
+      <Text style={styles.text}>Very high</Text>
```
Comment (tomekzaw) [inline app/src/examples/VolumeExample.tsx:137]: plz use `map`

**Review by tomekzaw — CHANGES_REQUESTED** [review]

File: app/src/examples/VolumeExample.tsx (hunk 3 above)
Comment (kacperkapusciak) [reply app/src/examples/VolumeExample.tsx:83]: Yup, still errors out 👍

File: app/src/examples/VolumeExample.tsx (hunk 4 above)
Comment (kacperkapusciak) [reply app/src/examples/VolumeExample.tsx]: Okay! Fixed in eade45b58

File: app/src/examples/VolumeExample.tsx (hunk 6 above)
Comment (kacperkapusciak) [reply app/src/examples/VolumeExample.tsx]: oki doki. Fixed in eade45b58

File: app/src/examples/VolumeExample.tsx [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+    borderRadius: 16,
+    borderWidth: 1,
+    width: 170,
+    height: 120,
+    alignSelf: 'center',
+    justifyContent: 'center',
+  },
+  value: {
+    textAlign: 'center',
+    // fontFamily: 'Poppins-Bold',
```
[the remark this answers is not in the document]
Comment (kacperkapusciak) [reply app/src/examples/VolumeExample.tsx]: 👍

File: app/src/examples/VolumeExample.tsx (hunk 5 above)
Comment (kacperkapusciak) [reply app/src/examples/VolumeExample.tsx:210]: Valid point but there are only two `Poppins-Bold` and one `Poppins-Regular`. If there would be more then I'd definitely do that but I think here it's fine to leave it as is

File: Example/ios/ReanimatedExample.xcodeproj/project.pbxproj [hunk 8]
```diff
@@ -552,6 +552,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = J5FM626PE2;
```
Comment (tomekzaw) [inline Example/ios/ReanimatedExample.xcodeproj/project.pbxproj:555]: plz revert

File: WebExample/package.json [hunk 9]
```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@expo/webpack-config": "^18.0.0",
+    "@react-native-community/slider": "^4.4.2",
```
Comment (tomekzaw) [inline WebExample/package.json:13]: according to https://docs.expo.dev/versions/latest/sdk/slider/, slider doesn't work on web so there's no point adding it as a dependency in WebExample I think

**Review by tomekzaw — CHANGES_REQUESTED** [review]

File: WebExample/package.json (hunk 9 above)
Comment (kacperkapusciak) [reply WebExample/package.json:13]: Nope, that's not right. It works on the web just fine
image
image

**Review by piaskowyk — APPROVED** [review]
🐶

**Review by tomekzaw — APPROVED** [review]

**Pull request merged**